action.yml

```diff
@@ -23,6 +23,10 @@ inputs:
     description: "Disables the sorting of list based on publish date"
     default: "false"
     required: false
+  reverse_sort:
```

Comment:
Rename to `sort_order` with default value `forward`. 

Values supported:
- forward
- reverse

Reply:
Addressed in b272c3e24a68592b58c8bd492fe357bd65295f6b